packages/clerk-js/src/utils/url.ts

```diff
@@ -231,6 +231,21 @@ export const appendAsQueryParams = (
   return base + (params.toString() ? '#/?' + params.toString() : '');
 };
 
+export const appendAsQueryParams = (
```

Comment:
❓Now that you kept the signature the same but changed the implementation, can you check if there is any wrong usage of this function ?

Reply:
I did check!